resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -385,13 +385,16 @@ private[spark] object Config extends Logging {
   val KUBERNETES_VOLUMES_HOSTPATH_TYPE = "hostPath"
   val KUBERNETES_VOLUMES_PVC_TYPE = "persistentVolumeClaim"
   val KUBERNETES_VOLUMES_EMPTYDIR_TYPE = "emptyDir"
+  val KUBERNETES_VOLUMES_NFS_TYPE = "nfs"
   val KUBERNETES_VOLUMES_MOUNT_PATH_KEY = "mount.path"
   val KUBERNETES_VOLUMES_MOUNT_SUBPATH_KEY = "mount.subPath"
   val KUBERNETES_VOLUMES_MOUNT_READONLY_KEY = "mount.readOnly"
   val KUBERNETES_VOLUMES_OPTIONS_PATH_KEY = "options.path"
   val KUBERNETES_VOLUMES_OPTIONS_CLAIM_NAME_KEY = "options.claimName"
   val KUBERNETES_VOLUMES_OPTIONS_MEDIUM_KEY = "options.medium"
   val KUBERNETES_VOLUMES_OPTIONS_SIZE_LIMIT_KEY = "options.sizeLimit"
+  val KUBERNETES_VOLUMES_OPTIONS_SERVER_KEY = "options.server"
+  val KUBERNETES_VOLUMES_OPTIONS_READ_ONLY_KEY = "options.readOnly"
```

Comment:
Do we need this option additionally? Then, could you add some comment about all combinations, please?
1. mount.readOnly=true, options.readOnly=true
2. mount.readOnly=true, options.readOnly=false
3. mount.readOnly=false, options.readOnly=true
4. mount.readOnly=false, options.readOnly=false

Reply:
In the above, `options.readOnly=None` is added because we don't failure at missing `options.readOnly`.

Reply:
I've removed the options.readOnly case, as you suggested.